sdk/keyvault/azure-keyvault-keys/samples/hello_world.py

```diff
@@ -34,11 +34,13 @@
 # 5. Delete a key (begin_delete_key)
 # ----------------------------------------------------------------------------------------------------------
 
+# [START create_a_key_client]
 # Instantiate a key client that will be used to call the service.
 # Here we use the DefaultAzureCredential, but any azure-identity credential can be used.
```

Comment:
Since READMEs most likely have context about credentials and method use preceding the snippets, I would think we can exclude sample file comments from the snippets that are captured